environment.yml

```diff
@@ -1,8 +1,5 @@
 # This file should have all the dependencies for development excluding the specific to the backends.
-name: ibis-dev
-channels:
-  - nodefaults
-  - conda-forge
+name: ibis
```

Comment:
I don't care that much about the name of the environment. `ibis-dev` was based on the pandas environemnt, which I assumed was named to avoid conflicts with `pandas` named environments created by devs. But I do think it's a poor open source development practice to make arbitrary changes based on personal preferences, and I would avoid them. It would consume a lot of everybody's energy if we keep changing things based on personal preferences.

Reply:
Happy to revert this back, I don't really care what it's called.

Reply:
It seemed more obvious that an environment would carry the name of the library but maybe that's just me.